2023-01-09T04:56:51.663762209Z I0109 04:56:51.663638       1 readyz.go:155] Listening on 0.0.0.0:9980
2023-01-09T04:56:54.587116546Z I0109 04:56:54.587079       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:56:54.588603465Z 2023/01/09 04:56:54 WARNING: [core] [Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:56:54.588603465Z   "Addr": "localhost:2379",
2023-01-09T04:56:54.588603465Z   "ServerName": "localhost",
2023-01-09T04:56:54.588603465Z   "Attributes": null,
2023-01-09T04:56:54.588603465Z   "BalancerAttributes": null,
2023-01-09T04:56:54.588603465Z   "Type": 0,
2023-01-09T04:56:54.588603465Z   "Metadata": null
2023-01-09T04:56:54.588603465Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:56:55.590084283Z 2023/01/09 04:56:55 WARNING: [core] [Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:56:55.590084283Z   "Addr": "localhost:2379",
2023-01-09T04:56:55.590084283Z   "ServerName": "localhost",
2023-01-09T04:56:55.590084283Z   "Attributes": null,
2023-01-09T04:56:55.590084283Z   "BalancerAttributes": null,
2023-01-09T04:56:55.590084283Z   "Type": 0,
2023-01-09T04:56:55.590084283Z   "Metadata": null
2023-01-09T04:56:55.590084283Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:56:56.587646308Z W0109 04:56:56.587604       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T04:56:58.589424595Z I0109 04:56:58.589383       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:56:58.591417854Z 2023/01/09 04:56:58 WARNING: [core] [Channel #3 SubChannel #4] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:56:58.591417854Z   "Addr": "localhost:2379",
2023-01-09T04:56:58.591417854Z   "ServerName": "localhost",
2023-01-09T04:56:58.591417854Z   "Attributes": null,
2023-01-09T04:56:58.591417854Z   "BalancerAttributes": null,
2023-01-09T04:56:58.591417854Z   "Type": 0,
2023-01-09T04:56:58.591417854Z   "Metadata": null
2023-01-09T04:56:58.591417854Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:56:59.587031818Z I0109 04:56:59.586983       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:56:59.587962721Z 2023/01/09 04:56:59 WARNING: [core] [Channel #5 SubChannel #6] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:56:59.587962721Z   "Addr": "localhost:2379",
2023-01-09T04:56:59.587962721Z   "ServerName": "localhost",
2023-01-09T04:56:59.587962721Z   "Attributes": null,
2023-01-09T04:56:59.587962721Z   "BalancerAttributes": null,
2023-01-09T04:56:59.587962721Z   "Type": 0,
2023-01-09T04:56:59.587962721Z   "Metadata": null
2023-01-09T04:56:59.587962721Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:56:59.592495377Z 2023/01/09 04:56:59 WARNING: [core] [Channel #3 SubChannel #4] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:56:59.592495377Z   "Addr": "localhost:2379",
2023-01-09T04:56:59.592495377Z   "ServerName": "localhost",
2023-01-09T04:56:59.592495377Z   "Attributes": null,
2023-01-09T04:56:59.592495377Z   "BalancerAttributes": null,
2023-01-09T04:56:59.592495377Z   "Type": 0,
2023-01-09T04:56:59.592495377Z   "Metadata": null
2023-01-09T04:56:59.592495377Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:00.588910718Z 2023/01/09 04:57:00 WARNING: [core] [Channel #5 SubChannel #6] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:00.588910718Z   "Addr": "localhost:2379",
2023-01-09T04:57:00.588910718Z   "ServerName": "localhost",
2023-01-09T04:57:00.588910718Z   "Attributes": null,
2023-01-09T04:57:00.588910718Z   "BalancerAttributes": null,
2023-01-09T04:57:00.588910718Z   "Type": 0,
2023-01-09T04:57:00.588910718Z   "Metadata": null
2023-01-09T04:57:00.588910718Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:00.591015939Z W0109 04:57:00.590996       1 etcdcli_pool.go:73] could not create a new cached client after 1 tries, trying again. Err: context deadline exceeded
2023-01-09T04:57:01.588052257Z W0109 04:57:01.588007       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T04:57:03.588429984Z I0109 04:57:03.588384       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:57:03.589396791Z 2023/01/09 04:57:03 WARNING: [core] [Channel #7 SubChannel #8] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:03.589396791Z   "Addr": "localhost:2379",
2023-01-09T04:57:03.589396791Z   "ServerName": "localhost",
2023-01-09T04:57:03.589396791Z   "Attributes": null,
2023-01-09T04:57:03.589396791Z   "BalancerAttributes": null,
2023-01-09T04:57:03.589396791Z   "Type": 0,
2023-01-09T04:57:03.589396791Z   "Metadata": null
2023-01-09T04:57:03.589396791Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:04.588044634Z I0109 04:57:04.587990       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:57:04.588934297Z 2023/01/09 04:57:04 WARNING: [core] [Channel #9 SubChannel #10] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:04.588934297Z   "Addr": "localhost:2379",
2023-01-09T04:57:04.588934297Z   "ServerName": "localhost",
2023-01-09T04:57:04.588934297Z   "Attributes": null,
2023-01-09T04:57:04.588934297Z   "BalancerAttributes": null,
2023-01-09T04:57:04.588934297Z   "Type": 0,
2023-01-09T04:57:04.588934297Z   "Metadata": null
2023-01-09T04:57:04.588934297Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:04.590393496Z 2023/01/09 04:57:04 WARNING: [core] [Channel #7 SubChannel #8] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:04.590393496Z   "Addr": "localhost:2379",
2023-01-09T04:57:04.590393496Z   "ServerName": "localhost",
2023-01-09T04:57:04.590393496Z   "Attributes": null,
2023-01-09T04:57:04.590393496Z   "BalancerAttributes": null,
2023-01-09T04:57:04.590393496Z   "Type": 0,
2023-01-09T04:57:04.590393496Z   "Metadata": null
2023-01-09T04:57:04.590393496Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:04.591443562Z I0109 04:57:04.591426       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:57:04.592166303Z 2023/01/09 04:57:04 WARNING: [core] [Channel #11 SubChannel #12] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:04.592166303Z   "Addr": "localhost:2379",
2023-01-09T04:57:04.592166303Z   "ServerName": "localhost",
2023-01-09T04:57:04.592166303Z   "Attributes": null,
2023-01-09T04:57:04.592166303Z   "BalancerAttributes": null,
2023-01-09T04:57:04.592166303Z   "Type": 0,
2023-01-09T04:57:04.592166303Z   "Metadata": null
2023-01-09T04:57:04.592166303Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:05.589566023Z W0109 04:57:05.589516       1 etcdcli_pool.go:73] could not create a new cached client after 1 tries, trying again. Err: context deadline exceeded
2023-01-09T04:57:05.589976225Z 2023/01/09 04:57:05 WARNING: [core] [Channel #9 SubChannel #10] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:05.589976225Z   "Addr": "localhost:2379",
2023-01-09T04:57:05.589976225Z   "ServerName": "localhost",
2023-01-09T04:57:05.589976225Z   "Attributes": null,
2023-01-09T04:57:05.589976225Z   "BalancerAttributes": null,
2023-01-09T04:57:05.589976225Z   "Type": 0,
2023-01-09T04:57:05.589976225Z   "Metadata": null
2023-01-09T04:57:05.589976225Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:05.593502702Z 2023/01/09 04:57:05 WARNING: [core] [Channel #11 SubChannel #12] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:05.593502702Z   "Addr": "localhost:2379",
2023-01-09T04:57:05.593502702Z   "ServerName": "localhost",
2023-01-09T04:57:05.593502702Z   "Attributes": null,
2023-01-09T04:57:05.593502702Z   "BalancerAttributes": null,
2023-01-09T04:57:05.593502702Z   "Type": 0,
2023-01-09T04:57:05.593502702Z   "Metadata": null
2023-01-09T04:57:05.593502702Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:06.588690486Z W0109 04:57:06.588645       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T04:57:06.592847196Z W0109 04:57:06.592804       1 etcdcli_pool.go:73] could not create a new cached client after 2 tries, trying again. Err: context deadline exceeded
2023-01-09T04:57:08.212963143Z I0109 04:57:08.212913       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:57:08.213780306Z 2023/01/09 04:57:08 WARNING: [core] [Channel #13 SubChannel #14] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:08.213780306Z   "Addr": "localhost:2379",
2023-01-09T04:57:08.213780306Z   "ServerName": "localhost",
2023-01-09T04:57:08.213780306Z   "Attributes": null,
2023-01-09T04:57:08.213780306Z   "BalancerAttributes": null,
2023-01-09T04:57:08.213780306Z   "Type": 0,
2023-01-09T04:57:08.213780306Z   "Metadata": null
2023-01-09T04:57:08.213780306Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:08.589264326Z I0109 04:57:08.589225       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:57:08.591385727Z 2023/01/09 04:57:08 WARNING: [core] [Channel #15 SubChannel #16] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:08.591385727Z   "Addr": "localhost:2379",
2023-01-09T04:57:08.591385727Z   "ServerName": "localhost",
2023-01-09T04:57:08.591385727Z   "Attributes": null,
2023-01-09T04:57:08.591385727Z   "BalancerAttributes": null,
2023-01-09T04:57:08.591385727Z   "Type": 0,
2023-01-09T04:57:08.591385727Z   "Metadata": null
2023-01-09T04:57:08.591385727Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:09.218407492Z 2023/01/09 04:57:09 WARNING: [core] [Channel #13 SubChannel #14] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:09.218407492Z   "Addr": "localhost:2379",
2023-01-09T04:57:09.218407492Z   "ServerName": "localhost",
2023-01-09T04:57:09.218407492Z   "Attributes": null,
2023-01-09T04:57:09.218407492Z   "BalancerAttributes": null,
2023-01-09T04:57:09.218407492Z   "Type": 0,
2023-01-09T04:57:09.218407492Z   "Metadata": null
2023-01-09T04:57:09.218407492Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:09.588481332Z I0109 04:57:09.588438       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:57:09.589581482Z 2023/01/09 04:57:09 WARNING: [core] [Channel #17 SubChannel #18] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:09.589581482Z   "Addr": "localhost:2379",
2023-01-09T04:57:09.589581482Z   "ServerName": "localhost",
2023-01-09T04:57:09.589581482Z   "Attributes": null,
2023-01-09T04:57:09.589581482Z   "BalancerAttributes": null,
2023-01-09T04:57:09.589581482Z   "Type": 0,
2023-01-09T04:57:09.589581482Z   "Metadata": null
2023-01-09T04:57:09.589581482Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:09.589607628Z I0109 04:57:09.589595       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:57:09.590895741Z 2023/01/09 04:57:09 WARNING: [core] [Channel #19 SubChannel #20] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:09.590895741Z   "Addr": "localhost:2379",
2023-01-09T04:57:09.590895741Z   "ServerName": "localhost",
2023-01-09T04:57:09.590895741Z   "Attributes": null,
2023-01-09T04:57:09.590895741Z   "BalancerAttributes": null,
2023-01-09T04:57:09.590895741Z   "Type": 0,
2023-01-09T04:57:09.590895741Z   "Metadata": null
2023-01-09T04:57:09.590895741Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:09.592385304Z 2023/01/09 04:57:09 WARNING: [core] [Channel #15 SubChannel #16] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:09.592385304Z   "Addr": "localhost:2379",
2023-01-09T04:57:09.592385304Z   "ServerName": "localhost",
2023-01-09T04:57:09.592385304Z   "Attributes": null,
2023-01-09T04:57:09.592385304Z   "BalancerAttributes": null,
2023-01-09T04:57:09.592385304Z   "Type": 0,
2023-01-09T04:57:09.592385304Z   "Metadata": null
2023-01-09T04:57:09.592385304Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:10.214130951Z W0109 04:57:10.214085       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T04:57:10.590539663Z W0109 04:57:10.590492       1 etcdcli_pool.go:73] could not create a new cached client after 1 tries, trying again. Err: context deadline exceeded
2023-01-09T04:57:10.590914949Z 2023/01/09 04:57:10 WARNING: [core] [Channel #17 SubChannel #18] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:10.590914949Z   "Addr": "localhost:2379",
2023-01-09T04:57:10.590914949Z   "ServerName": "localhost",
2023-01-09T04:57:10.590914949Z   "Attributes": null,
2023-01-09T04:57:10.590914949Z   "BalancerAttributes": null,
2023-01-09T04:57:10.590914949Z   "Type": 0,
2023-01-09T04:57:10.590914949Z   "Metadata": null
2023-01-09T04:57:10.590914949Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:10.591354081Z 2023/01/09 04:57:10 WARNING: [core] [Channel #19 SubChannel #20] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:57:10.591354081Z   "Addr": "localhost:2379",
2023-01-09T04:57:10.591354081Z   "ServerName": "localhost",
2023-01-09T04:57:10.591354081Z   "Attributes": null,
2023-01-09T04:57:10.591354081Z   "BalancerAttributes": null,
2023-01-09T04:57:10.591354081Z   "Type": 0,
2023-01-09T04:57:10.591354081Z   "Metadata": null
2023-01-09T04:57:10.591354081Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:57:11.589450007Z W0109 04:57:11.589407       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T04:57:11.590516748Z W0109 04:57:11.590482       1 etcdcli_pool.go:73] could not create a new cached client after 2 tries, trying again. Err: context deadline exceeded
2023-01-09T04:57:12.214256838Z I0109 04:57:12.214212       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:57:14.590788886Z I0109 04:57:14.590744       1 etcdcli_pool.go:70] creating a new cached client
